### PR TITLE
Kellychanges

### PR DESCRIPTION
**Pull request by tangkelly1999**
I made more changes. How do i format the pictures better it looks weird when i change the screen size. where do you think i should put like a link for my resume. Any suggestions/comments to style and layout would be quite valuable. Thanks bud!

### CODE REVIEW

**Comment by tangkelly1999** [comment]
also pls help me make my description sound more like i know stuff :)

**Review by andyjiin — CHANGES_REQUESTED** [review]
First of all, why do you have a bunch of random screenshots. And secondly I would make the images much bigger. And do you like the font?

File: main.html [hunk 1]
```diff
@@ -40,35 +42,120 @@
...
+     <header class="body" d-flex" style="overflow-x: hidden; overflow-y: hidden;">
+       <div class="overlay" style="position: absolute;"></div>
+     </header>
+
     <section class="home p-3 p-lg-5 d-flex d-column" id="home">
       <div class="my-auto">
         <h1 class="mb-0">Kelly Tang</h1>
-        <div class="subheading mb-5"> Electrical Engineering Student </div>
-        <p class="lead mb-5">I know how to solder hehe xd</p>
+        <div class="subheading mb-5"> University of Waterloo | Electrical Engineering Student </div>
```
Comment (andyjiin) [inline main.html]: nice flex

File: main.html [hunk 2]
```diff
@@ -40,35 +42,120 @@
...
+
+ 
+
     </section>
 
     <section class="content p-3 p-lg-5 d-flex flex-column" id="about">
       <div>
         <h2 class="mb-5">About</h2>
-        Once upon a time, I woke up on time. im a loser
+        Hello there! I'm Kelly and I'm an engineering student at the University of Waterloo. My design experience ranges from PCB design to power distribution (grammar?). I'm interested in ???. Currently looking for job opportunities for fall 2019. pLS hire.
```
Comment (andyjiin) [inline main.html]: bad change I like what it said before hehe. But this page is very empty we can work on it later

File: stylesheet.css [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
+  background-position: center 0;
+  background-repeat: no-repeat;
+  background-size: 100% auto;
+
+
 }
 
+
 h1, h2, h3, h4, h5, h6 {
+  color: #FFF0F5;
```
Comment (andyjiin) [inline stylesheet.css:20]: You can barely see the title colours are you going to change the background color as well?

File: main.html [hunk 4]
```diff
@@ -40,35 +42,120 @@
...
       </div>
+    </section><!---END PROJECTS---->
+
+    <!----MODAL POPUP--->
+
+    <section class="modal fade" id="DriverDisplayModal" aria-hidden="true">
+    	<div class="modal-dialog modal-lg modal-dialog-centered ">
+    		<div class="modal-content">
+    			<div class="modal-header">
+    				<button class="close" type="button" data-dismiss="modal">x</button>
```
Comment (andyjiin) [inline main.html]: This button looks kinda ugly I don't think you really need it but maybe you can change it to an `up-arrow` or something

**Review by andyjiin — CHANGES_REQUESTED** [review]

**Pull request merged**